Makefile

```diff
@@ -108,7 +108,7 @@ pkg/apis/integreatly/v1alpha1/zz_generated.deepcopy.go:	pkg/apis/integreatly/v1a
 
 .PHONY: code/gen
 code/gen: setup/moq deploy/crds/integreatly.org_rhmis_crd.yaml pkg/apis/integreatly/v1alpha1/zz_generated.deepcopy.go pkg/apis/integreatly/v1alpha1/zz_generated.openapi.go
-	find ./ -name *_moq.go -type f -not -path "./vendor/*" -delete
+	# find ./ -name *_moq.go -type f -not -path "./vendor/*" -delete
```

Comment:
any reason not to delete this? No need to hold up the PR or anything, just curious

Reply:
yes I think it can be deleted. @grdryn do you recall what problem this one was causing that we removed it?